app/models/application_qualification.rb

```diff
@@ -59,6 +59,7 @@ class ApplicationQualification < ApplicationRecord
 
   validates :qualification_type, length: { maximum: MAX_QUALIFICATION_TYPE_LENGTH }, allow_blank: true
   validates :non_uk_qualification_type, length: { maximum: MAX_QUALIFICATION_TYPE_LENGTH }, allow_blank: true
+  validates :enic_reason, presence: true
```

Comment:
Will there be a small window where the data migration has not run yet but this can cause errors until it will run?

I'm basically asking if shipping this validation after migrating the data, in a separate PR is worth doing or not.

Reply:
Agree